examples/strong_branching_portfolio.jl

```diff
@@ -11,10 +11,9 @@ import HiGHS
 
 
 # For bug hunting:
-seed = rand(UInt64)
```

Comment:
should still be left here for bug hunting

Reply:
Yes, but it is easier to track if all tests and examples use the same seed set in `runtest.jl`. If we want to test individuals files, you can comment this part in.